liquidity/liquidity.go

```diff
@@ -457,6 +499,160 @@ func (m *Manager) ForceAutoLoop(ctx context.Context) error {
 	}
 }
 
+// dispatchBestEasyAutoloopSwap tries to dispatch a swap to bring the hot wallet
+// size back to its target.
+func (m *Manager) dispatchBestEasyAutoloopSwap(ctx context.Context) error {
+	// If autoloop is turned off, return early.
+	if m.params.Autoloop == false {
+		return nil
+	}
+
+	// Retrieve existing swaps.
+	loopOut, err := m.cfg.ListLoopOut()
+	if err != nil {
+		return err
+	}
+
+	loopIn, err := m.cfg.ListLoopIn()
+	if err != nil {
+		return err
+	}
+
+	// Get a summary of our existing swaps so that we can check our autoloop
+	// budget.
+	summary, err := m.checkExistingAutoLoops(ctx, loopOut, loopIn)
+	if err != nil {
+		return err
+	}
+
+	if summary.totalFees() >= m.params.AutoFeeBudget {
```

Comment:
Given these checks are the same we already do in `SuggestSwaps` perhaps worth creating helper functions for them to not duplicate code and logs.